transbank/oneclick/transaction_mall.py

```diff
@@ -0,0 +1,79 @@
+import requests
+
+from transbank.common.headers_builder import HeadersBuilder
+from transbank.common.integration_type import IntegrationType, webpay_host
+from transbank.common.options import Options, WebpayOptions
+from transbank import oneclick
+from transbank.error.transaction_authorize_error import TransactionAuthorizeError
+from transbank.error.transaction_refund_error import TransactionRefundError
+from transbank.error.transaction_status_error import TransactionStatusError
+from transbank.oneclick.request import TransactionAuthorizeRequest, TransactionRefundRequest, \
+    MallTransactionAuthorizeDetails
+from transbank.oneclick.response import TransactionAuthorizeResponse, TransactionRefundResponse, \
+    TransactionStatusResponse
+from transbank.oneclick.schema import TransactionAuthorizeRequestSchema, TransactionAuthorizeResponseSchema, \
+    TransactionRefundRequestSchema, TransactionRefundResponseSchema, TransactionStatusResponseSchema
+
+
+class Transaction(object):
```

Comment:
The class name should be MallTransaction

Reply:
done